zeebe/test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -122,7 +123,9 @@ public class CompactRecordLogger {
           entry("ROLE", "RL"),
           entry("GROUP", "GR"),
           entry("MAPPING", "MAP"),
-          entry("ASYNC_REQUEST", "ASYNC"));
+          entry("ASYNC_REQUEST", "ASYNC"),
+          entry("MULTI_INSTANCE", "MI"),
+          entry("INPUT_COLLECTION_EVALUATED", "IN_COL_EVAL"));
```

Comment:
🙃 I wonder if `IN_COL_EVAL` would be clear enough for a new engineer that sees this shorthand for the first time. Do you think `MI_INPUT_EVAL` would be clearer and still correct?